docs/reference/pipeline_schedules.md

```diff
@@ -12,7 +12,17 @@ Additionally, under a `variables` key you can add the pipeline schedule variable
 
     * Do not set `description` attribute - see [#535](https://github.com/gitlabform/gitlabform/issues/535#issue-1678509984)
 
-Example:
+There are 2 gitlabform specific keys/configs that can be set under `schedules` or individual schedule:
+
+- `delete` - set this to `true` under a specific schedule to delete that particular schedule.
+- `enforce` - set this to `true` under `schedules` so that any schedules that are not in `schedules` section are deleted.
+
+    !!! warning
+
+        The `delete` key maybe removed in future version of gitlabform as `enforce` key can be used for deleteing unconfigured schedules and keep gitlabform config cleaner.
```

Comment:
I am not sure if we should declare this. I don't remember, have we decided that we want to deprecate `delete`? I personally rather like it, it's like Puppet's `ensure: absent` pattern.

Reply:
Ahh... You're right! We didn't discuss about deprecating `delete`. I guess I just made an assumption because I didn't see a need for it anymore. This warning was kind of a way to start the conversation 😛 .

We can keep `delete` alongside `enforce` in the future, if you think it's going to be useful.

Reply:
I've removed this warning from the doc in latest commit.

Reply:
🙏 ☺️